InternalDocs/README.md

```diff
@@ -11,13 +11,16 @@ The core dev team attempts to keep this documentation up to date. If
 it is not, please report that through the
 [issue tracker](https://github.com/python/cpython/issues).
 
```

Comment:
Thanks for doing this. I think before we move this to the new area we need to make sure that it's all up to date, and that it is documenting the current state of affairs.

On line 9 it says "This document proposes..."  this should now be documentation, not a proposal. "These tools would be used to..." we need to document what they are used for. (This is not the place to discuss ideas for future plans, unless it's to explain some future-proofing design choices.)

Reply:
Of course, I understand that this is a place for finalized internal documentation.
I've just wanted to read this docs for personal understanding and have found some small issues.

Reply:
Sure, if you want to continue editing this into documentation form then we can do that on this PR.

Reply:
On the other hand, if you just want to make minor fixes in this PR that's fine, but leave the doc where it is.

Reply:
That's great. Yes, we will review.